docs/source/installation.rst

```diff
@@ -39,9 +41,7 @@ VulnerableCode is distributed with ``Dockerfile`` and ``docker-compose.yml`` fil
 required for the creation of the Docker image.
 
 Clone the git `VulnerableCode repo <https://github.com/nexB/vulnerablecode>`_,
-create an environment file, and build the Docker image:
-
-.. code-block:: bash
+create an environment file, and build the Docker image::
 
```

Comment:
Is there a reason to prefer ``::`` over ``code-block`` ?

Reply:
This makes this more readable when looking at the plain text ... it has the same effect. Less is more maybe?